app/renderer/components/navigation/navigator.js

```diff
@@ -149,6 +151,158 @@ class Navigator extends React.Component {
     ipc.off(messages.SHORTCUT_ACTIVE_FRAME_FORWARD, this.onForward)
   }
 
+  // BEM Level: navbarCaptionButtonContainer__navbarMenubarFlexContainer__navigatorWrapper
+  get navigatorWrapper () {
+    return <div className={cx({
+      navigatorWrapper: true,
+      [css(styles.navigatorWrapper)]: true
+    })}
+      onDoubleClick={this.onDoubleClick}
+      onDragOver={this.onDragOver}
+      onDrop={this.onDrop}
+    >
+      {this.topLevelStartButtons}
+      <NavigationBar />
+      {this.topLevelEndButtons}
+    </div>
+  }
+
+  // BEM Level: navbarCaptionButtonContainer__navbarMenubarFlexContainer__navigatorWrapper__topLevelStartButtons
```

Comment:
creating components without knowing which level you are on the hierarchy will lead to style inconsistency, which is all about we are trying to avoid.

Yet because the comment is just an option, I think it's not the perfect way, though it should work.